lib_eio/process.ml

```diff
@@ -84,12 +84,13 @@ let spawn ~sw (t:#mgr) ?cwd ?stdin ?stdout ?stderr ?env ?executable args =
     ?stdout:(stdout :> Flow.sink option)
     ?stderr:(stderr :> Flow.sink option)
 
-let run (t:#mgr) ?cwd ?stdin ?stdout ?stderr ?env ?executable args =
+let run (t:#mgr) ?cwd ?stdin ?stdout ?stderr ?is_success ?env ?executable args =
```

Comment:
Just a minor thing, but it might be simpler to provide a default function rather than handling the option later:
```suggestion
let run (t:#mgr) ?cwd ?stdin ?stdout ?stderr ?(is_success=Int.equal 0) ?env ?executable args =
```

Reply:
Good idea